### DISCUSSION TITLE
Since 3.48.2 often 'Unable to move file to trash'

### DISCUSSION THREAD

**Discussion by fumakotaro** [mifi/lossless-cut#1383] [Support]
As the title says, I now often get the message that LosslessCut was unable to move files to the trash and I am asked if I want to permanently delete them (which works).

There is no error code or anything, I remember having a similar problem in earlier versions where it said something like 'EBUSY: File opened somewhere else' but this is not the case here.

EDIT:
Before, this thread said 'always' but I changed it to 'often' since it actually worked now on a similar named file in the same folder.

**Comment by fumakotaro** [comment]
Now the following happened: 
In the same instance of LosslessCut that already successfully put files into the trash, I opened a new file from the same source folder with a similar name. I didn't make any segments or exports and then clicked the **Close file and clean up** button.
Again it said **Unable to move file to trash - Do you want to permanently delete it?** – I clicked yes.
Then an error popup showed: **Unable to delete file: ENOENT: no such file or directory, unlink 'C:\bla\bla\bla-proj.llc'**

But it did actually put bla.mp4 in the Windows Recycle Bin! Not the bla-proj.llc though this seems to have been permanently deleted as it is not in the working dir either.

Edit: This error happens all the time when I want to trash a file without making any segments/exports. Maybe it wants to remove the -proj.llc which doesn't exist because I didn't make any segments?

**Comment by mifi** [comment]
Is this the same as #1381?

**Reply by fumakotaro** [reply]
Yes! Sorry for getting back so late.

**Reply by mifi** [reply]
It should be fixed now
